_includes/moto.md

```diff
@@ -0,0 +1,90 @@
+{% if include.api_resource == "creditcard" %}
+    {% assign api_resource_field_name = "payment" %}
+{% else %}
+    {% assign api_resource_field_name = "paymentorder" %}
+{% endif %}
+
+## MOTO
+
+{% include alert-agreement-required.md %}
+
+MOTO (Mail Order / Telephone Order) is a purchase where you as a merchant enter 
+the customer card details in order to make a payment. The card details are given
```

Comment:
```suggestion
the payer's card details in order to make a payment. The card details are given
```
Just a bit of nitpicking to avoid confusion :) In the developer portal, we use the term _payer_ for _consumer_ / _end user_ / payer_. (Internally, _customer_ is the term used for _merchant_, and consumer is the term used for _payer/consumer_.)